query-engine/js-connectors/src/queryable.rs

```diff
@@ -125,16 +126,16 @@ impl QuaintQueryable for JsBaseQueryable {
 }
 
 impl JsBaseQueryable {
-    async fn build_query(sql: &str, values: &[quaint::Value<'_>]) -> Query {
+    async fn build_query(sql: &str, values: &[quaint::Value<'_>]) -> quaint::Result<Query> {
         let sql: String = sql.to_string();
-        let args = values.iter().map(|v| v.clone().into()).collect();
-        Query { sql, args }
+        let args = conversion::conv_params(values)?;
```

Comment:
We were totally missing the Quaint -> Driver-specific parameter conversion that e.g. `mysql` does [here](https://github.com/prisma/prisma-engines/blob/0bc8ee687eb17771fa7ca5167bcad38f8f2d0847/quaint/src/connector/mysql.rs#L488), before running the actual parametrised query.